### PR TITLE
[daint-gpu] Suggest EB module fix w.r.t lmod

### PR DESCRIPTION
**Pull request by kraushm**
This is a suggestion for a fix, which became necessary for the lmod installed on Tsa (tsa-cn017/cn018).

### CODE REVIEW

File: easybuild/module/EasyBuild-custom/cscs [hunk 1]
```diff
@@ -145,6 +145,7 @@ if { ! [ info exists ::env(EASYBUILD_SOURCEPATH) ] } {
 # Adding easybuild installed software to the list of modules
 prepend-path MODULEPATH                         $::env(EASYBUILD_INSTALLPATH)/modules/all
 # Loading easybuild module
+module use /apps/common/UES/easybuild/modules/all
 module load /apps/common/UES/easybuild/modules/all/EasyBuild/latest
```
Comment (gppezzi) [inline easybuild/module/EasyBuild-custom/cscs]:
```suggestion
module load EasyBuild/latest
```

**Comment by gppezzi** [comment]
Just for the record: the reason behind this PR is that Lmod in tsa doesn't like to load a module using an absolute path as we were doing so far for EasyBuild.

**Comment by gppezzi** [comment]
If we add this new `module use /apps/common/UES/easybuild/modules/all` statement, we could eliminate the 'latest' symlink (which sometimes is confusing), define a default for eb (the modules way) and simply do:
```
module load EasyBuild
``` 

How about that @lucamar @victorusu @kraushm ?

**Comment by lucamar** [comment]
@gppezzi The modules under `/apps/common/UES/easybuild/modules/all/EasyBuild` are created by EasyBuild `bootstrap.py`, e.g. `/apps/common/UES/easybuild/bootstrap_eb.py-3.9.1`: is there any option to set a module as default running `bootstrap`? I only see the following options
```
usage: bootstrap_eb.py-3.9.1 [-h] prefix

positional arguments:
  prefix      Installation prefix directory

optional arguments:
  -h, --help  show this help message and exit
```

**Comment by gppezzi** [comment]
By "module way" I meant creating a `.version` file. I've just set the default to 3.9.1 for testing: 
```
EasyBuild/3.9.1(default)
```

**Comment by lucamar** [comment]
@gppezzi That's fine for me, we just need to manually update the `.version` file after each EasyBuild installation instead of updating the symlink

**Comment by kraushm** [comment]
> If we add this new `module use /apps/common/UES/easybuild/modules/all` statement, we could eliminate the 'latest' symlink (which sometimes is confusing), define a default for eb (the modules way) and simply do:
> 
> ```
> module load EasyBuild
> ```
> 
> How about that @lucamar @victorusu @kraushm ?

Tests on Tsa and Daint worked fine, therefore I commited the proposed change.

**Pull request merged**